test/e2e/page-objects/pages/header-navbar.ts

```diff
@@ -42,31 +37,32 @@ class HeaderNavbar {
   }
 
   async lockMetaMask(): Promise<void> {
-    await this.driver.clickElement(this.accountOptionMenu);
-    // fix race condition with mmi build
-    if (process.env.MMI) {
-      await this.driver.waitForSelector(this.mmiPortfolioButton);
-    }
+    await this.openAccountOptionMenu();
     await this.driver.clickElement(this.lockMetaMaskButton);
   }
 
   async openAccountMenu(): Promise<void> {
     await this.driver.clickElement(this.accountMenuButton);
   }
 
+  async openAccountOptionMenu(): Promise<void> {
+    console.log('Open account options menu');
+    await this.driver.clickElement(this.accountOptionMenu);
```

Comment:
`accountOptionMenu` is slightly confusing as there is also `accountMenuButton`.The main issue is that the data-testid that was defined is itself incorrectly named, as shown below. Correcting this would be a huge amount of work.
```
private readonly accountMenuButton = '[data-testid="account-menu-icon"]';

  private readonly accountOptionMenu =
    '[data-testid="account-options-menu-button"]';
```
I am thinking that adding `toggleThreeDotMenu`, `openThreeDotMenu`, or `showThreeDotMenu `would be easier, but I leave it to you to decide as you have worked on this more.

Reply:
Make sense, i renamed to `openThreeDotMenu`